examples/remote_procedure/toxicity_detection/hydra_configs/conf/launch_with_local.yaml

```diff
@@ -0,0 +1,24 @@
+#@package _global_
+defaults:
+  - /mephisto/blueprint: remote_procedure
+  - /mephisto/architect: local
+  - /mephisto/provider: mock
+mephisto:
+  blueprint:
+    task_source: ${task_dir}/webapp/build/bundle.js
+    link_task_source: false
+    # NOTE pick something based on your task
+    block_qualification: test_qual_block
+    units_per_assignment: 1
+  task:
+    allowed_concurrent: 1
+    task_name: remote-procedure-test-task
+    task_title: "Provide feedback on our MNIST model"
+    # NOTE you'll want to update your task description
+    task_description: "You will draw digits. Try to fool our MNIST model, and provide us the correct label."
+    # NOTE set a reasonable reward
+    task_reward: 0.05
+    # NOTE will want real tags
+    task_tags: "mnist,drawing,models,correction"
+    # We expect to handle 25 people using the MNIST model at once
```

Comment:
we won't be using an mnist model for this one so comments will need to be updated